omnibus/config/software/datadog-agent.rb

```diff
@@ -61,12 +63,12 @@
     end
     command "inv -e rtloader.make --python-runtimes #{py_runtimes_arg} --install-prefix \"#{windows_safe_path(python_2_embedded)}\" --cmake-options \"-G \\\"Unix Makefiles\\\"\" --arch #{platform}", :env => env
     command "mv rtloader/bin/*.dll  #{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent/"
-    command "inv -e agent.build --exclude-rtloader --python-runtimes #{py_runtimes_arg} --major-version #{major_version_arg} --rebuild --no-development --embedded-path=#{install_dir}/embedded --arch #{platform} #{do_windows_sysprobe}", env: env
+    command "inv -e agent.build --exclude-rtloader --python-runtimes #{py_runtimes_arg} --major-version #{major_version_arg} --rebuild --no-development --embedded-path=#{install_dir}/embedded --arch #{platform} #{do_windows_sysprobe} --flavor #{flavor_arg}", env: env
```

Comment:
Is `AGENT_FLAVOR` environment variable always set here? Other places seem to check for it.

Reply:
So there's two sides to this:
* If invoked manually, the environment variable may not be present, but so is not `PY_RUNTIMES`, because we just expect this to be run with `inv -e agent.omnibus-build`.
* But the truth is that the `get_omnibus_env` function only sets it conditionally right now, so I'll make sure it always sets it, similar to `PY_RUNTIMES`.